src/main/java/com/labsynch/labseer/service/BulkLoadServiceImpl.java

```diff
@@ -825,9 +825,77 @@ public Parent validateParent(Parent parent, Collection<BulkLoadPropertyMappingDT
 							|| foundParent.getCorpName().contains(parent.getLabelPrefix().getLabelPrefix()));
 					if (sameStereoCategory & sameStereoComment
 							& (sameCorpName | (noCorpName & sameCorpPrefixOrNoPrefix))) {
-						// parents match
-						parent = foundParent;
-						break searchResultLoop;
+						// parents match (based on above criteria)
+						boolean equalAliases = true; // assumption is aliases are same unless proven to be false
+
+						// Need to Create String Sets to Do 
+						// "Is alias (str) of parent in aliases of foundParent (set of strs)?" (and vice versa) logic 
+
+						// Note: This is not the most efficient / concise code to do this process
+						// however due to readability and the assumingly small number of aliases 
+						// parents will have this is fine 
+
+						Set<String> parentAliasStrings = new HashSet<>();
+						for(ParentAlias alias : parent.getParentAliases()){
+							parentAliasStrings.add(alias.toString());
+						}
+						Set<String> foundParentAliasStrings = new HashSet<>(); 
+						for(ParentAlias alias : foundParent.getParentAliases()){
+							foundParentAliasStrings.add(alias.toString());
```

Comment:
What does this `alias.toString()` method return? Is it just the `aliasName`? Or is it an Object like `ParentAlias[lsType=null,lsKind=null,id=27,aliasName="my favorite alias"]`?

Reply:
I suppose `getAlias()` would be more appropriate. Also I forgot to add in if it is marked as ignored so I will do that as well.